package.json

```diff
@@ -48,9 +48,9 @@
   },
   "peerDependencies": {
     "graphql": "^0.6.0 || ^0.7.0 || ^0.8.0-b || ^0.9.0",
-    "prop-types": ">=15.5.10",
-    "react": "^0.14.9 || >=15.5.4",
-    "react-dom": "^0.14.9 || >=15.5.4"
+    "prop-types": ">=15.5.0",
+    "react": "^0.14.9 || >=15.0.0",
```

Comment:
We can't go lower than 15.3.0 here because "prop-types" [is documented as supporting](https://github.com/facebook/prop-types) 0.14.9 and 15.3.0+ only.